notebooks/text_models/solutions/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>* "to train a your BERT" -&gt; "to train a BERT"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>